lib/spender/money_logs/owner.ex

```diff
@@ -13,7 +13,7 @@ defmodule Spender.MoneyLogs.Owner do
   schema "owners" do
     field :name, :string
     field :type, :string
-    has_many :budgets, Budget, on_delete: :delete_all
+    has_many :moneylog, Moneylog, on_delete: :delete_all
```

Comment:
This too should be `moneylogs` since later you want to access them as `Owner.moneylogs`